.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: go
 go:
   - 1.7
+  - 1.8
```

Comment:
Could you change all to be the latest point releases, e.g. regression I've observed ~ `go tool -race` is 1.5x slower in [1.8](https://groups.google.com/forum/#!topic/golang-nuts/HbHBXFrpdoo) (fixed in 1.8.1 iirc)